opi/plugins/brave.py

```diff
@@ -19,5 +19,9 @@ def run(cls, query):
 			gpgkey = 'https://brave-browser-rpm-release.s3.brave.com/brave-core.asc'
 		)
 
+		# prevent post install script from messing with our repos
+		subprocess.call(['sudo', 'rm', '-f', '/etc/default/brave-browser'])
+		subprocess.call(['sudo', 'touch', '/etc/default/brave-browser'])
+
 		opi.install_packages(['brave-browser'])
-		opi.ask_keep_repo('brave-browser')
+		opi.ask_keep_repo('brave-browser')
```

Comment:
Please make sure that your files have a newline at the end of the file.
Your editor might not show the actual newline but eg vim complains with `[noeol]`.

Reply:
I changed my editor settings. It has the new line at the end now.